CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,255 @@
+
+   * A description of the person's behavior in neutral language
+   * The negative impact of that behavior
+   * A concrete behavioral modification plan
+   * Any consequences of their behavior
+The team should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the team can accept the apology on behalf of the reporter.
+
+
+
+## Decide further responses
+
+If the reported person provides additional context, the Manim Community Code of Conduct team may need to re-evaluate the behavioral modification plan and consequences.
+
+## Follow up with the reporter
+
+A person who makes a report should receive a follow up response stating what action was taken in response to the report. If the team decided no response was needed, they should provide an explanation why it was not a Code of Conduct violation. Reports that are not made in good faith (such as "reverse sexism" or "reverse racism") may receive no response.
+
+The follow up should be sent no later than one week after the receipt of the report. If deliberation or follow up with the reported person takes longer than one week, the team should send a status update to the reporter.
+
+
+
+## Changes to Code of Conduct
+
+When discussing a change to the Manim Community code of conduct or enforcement procedures, the Manim Community Code of Conduct team will follow this decision-making process:
+
+   * **Brainstorm options.** Team members should discuss any relevant context and brainstorm a set of possible options. It is important to provide constructive feedback without getting side-tracked from the main question.
+   * **Vote.** Proposed changes to the code of conduct will be decided by a two-thirds majority of all voting members of the Code of Conduct team. Team members are listed in the charter. Currently active voting members are listed in the following section.
+   * **Board Vote.** Once a working draft is in place for the Code of Conduct and procedures, the Code of Conduct team shall provide the Manim Community Moderators with a draft of the changes. The Manim Community Moderators will vote on the changes at a board meeting.
+
+
+## Current list of voting members
+
+- All available Community Developers (i.e. those with "write" permissions, or above, on the Manim Community GitHub organization).
+
+
+
+## License
+
+This Code of Conduct is licensed under the [Creative Commons Attribution-ShareAlike 3.0 Unported License](https://creativecommons.org/licenses/by-sa/3.0/).
+
+
+
+## Attributions
+
+This Code of Conduct was forked from the code of conduct from the [Python Software Foundation](https://www.python.org/psf/conduct/) and adapted by Manim Community.
```

Comment:
```suggestion
The Code of Conduct can be found at docs/source/conduct.rst and respectively at https://docs.manim.community/en/latest/conduct.html
```

Reply:
:-1: to this. Github will ask users to read this whenever it changes. So, this needs to be here. Maybe docs can have a symlink to this.

Reply:
Ah, good point!
Yess, then +1 for the docs and the symlink idea.

Reply:
What do we mean by symlink here? Remember symlinks don't work on windows (do they?)

Reply:
They don't work on windows yes, but atleast they work on RTD, and that's enough.

Reply:
That's what I thought. :+1:

Reply:
So what's the course of action? Am I to remove the content from conduct.rst and just simply put a link/href to CODE_OF_CONDUCT.md? I'm not sure what we mean by symlink here...

Reply:
Oh! Are we talking about renaming conduct.rst to conduct.md and making it a symlink to CODE_OF_CONDUCT.md since sphinx webpages are hosted on Linux servers? This way the content is updated in the docs when the CODE_OF_CONDUCT.md is updated indirectly?

Reply:
correct.